cf/commands/curl.go

```diff
@@ -54,7 +54,7 @@ func (cmd *Curl) MetaData() commandregistry.CommandMetadata {
    will be set to application/json. You may override headers with -H and the
    request method with -X.
 
-   For API documentation, please visit http://apidocs.cloudfoundry.org.`),
+   For API documentation, please visit http://v2-apidocs.cloudfoundry.org.`),
```

Comment:
curl can be for both v2 and v3 (depending on what the user enters), so should we modify the link here or add v2 link along with existing one?

Reply:
I think since the v3 is newer, and it includes a link to the v2 docs, we should leave this as `apidocs.cloudfoundry.org` for the same reason as the comment about the v3 `client`